src/solvers/refinement/string_refinement.cpp

```diff
@@ -1532,9 +1532,9 @@ class find_index_visitort: public const_expr_visitort
 /// (str[k] == 'a') and str, the function should return k
 /// \par parameters: a formula expr and a char array str
 /// \return an index expression
-exprt find_index(const exprt &expr, const exprt &str)
+exprt find_index(const exprt &expr, const exprt &str, const symbol_exprt &qvar)
```

Comment:
At the same time, the `parameters` line should be split into separate `\param` lines:
```
/// \param expr: a formula expr
/// \param str: a char array str
/// \param qvar: ???
```